enos/utils/extra.py

```diff
@@ -104,6 +104,8 @@ def get_kolla_required_values(env):
     #        "%s:%s" % (env['config']['registry']['ip'],
     #                   env['config']['registry']['port'])
 
+    values['openstack_release'] = OPENSTACK_RELEASE
```

Comment:
What if we want to override that value, such as in https://github.com/BeyondTheClouds/enos-scenarios/blob/master/osprofiler/reservation.yaml#L66

Reply:
This should be still possible according to : https://github.com/BeyondTheClouds/enos/blob/f4986c53b4c529f854ce9d798931713e9d303cb3/enos/utils/extra.py#L145-L148

and https://github.com/BeyondTheClouds/enos/blob/f4986c53b4c529f854ce9d798931713e9d303cb3/enos/utils/extra.py#L112

Reply:
:+1: